app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -17,43 +16,32 @@ import TableMetricPrimary from "../TableMetricPrimary";
 import TableMetricSecondary from "../TableMetricSecondary";
 import { AnalysisData } from "../../lib/visualization/types";
 
-const PageResults: React.FunctionComponent<RouteComponentProps> = () => {
-  const { slug } = useParams();
-  const { loading, error, result: analysis } = useAnalysis(slug);
-
-  return (
-    <AppLayoutWithExperiment title="Analysis" testId="PageResults">
-      {({ experiment }) => (
-        <>
-          {loading ? (
-            <p data-testid="analysis-loading">Loading analysis data...</p>
-          ) : (
-            <>
-              <h3 className="h5 mb-3">Monitoring</h3>
-              <p>
-                <LinkExternal
-                  href={experiment.monitoringDashboardUrl!}
-                  data-testid="link-monitoring-dashboard"
-                >
-                  Click here
-                </LinkExternal>{" "}
-                to view the live monitoring dashboard.
-              </p>
-              <h3 className="h5 mb-3 mt-4">Overview</h3>
-
-              {error && <AnalysisFetchError />}
-              {analysis?.show_analysis ? (
-                <AnalysisAvailable {...{ experiment, analysis }} />
-              ) : (
-                <AnalysisUnavailable {...{ experiment }} />
-              )}
-            </>
-          )}
-        </>
-      )}
-    </AppLayoutWithExperiment>
-  );
-};
+const PageResults: React.FunctionComponent<RouteComponentProps> = () => (
+  <AppLayoutWithExperiment title="Analysis" testId="PageResults">
+    {({ experiment, analysis }) => (
+      <>
+        <h3 className="h5 mb-3">Monitoring</h3>
+        <p>
+          {/* TODO: start/end dates with EXP-607 */}
```

Comment:
Hmm might need to rebase, looks like [these changes](https://github.com/mozilla/experimenter/pull/4070/files) would be overwritten?

Reply:
Whoops. Changed it back.